src/Directory.Build.props

```diff
@@ -0,0 +1,6 @@
+<Project>
+    <PropertyGroup>
+        <LangVersion>10</LangVersion>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+</Project>
```

Comment:
Looks like indentation is off (`4` spaces instead of `2` for XML?)

Reply:
I've added a shared `.editorconfig` and reformatted.